src/content-service/services/__tests__/api-provider.spec.ts

```diff
@@ -142,5 +146,25 @@ describe('Content service API', () => {
         });
       });
     });
+
+    it('should get image with default size', () => {
+      const productEntity = EntityType.Products;
+      const entityId = 'test';
+      const imagesBaseUrl = pathSettings[Environment.Dev].images;
+      const images = getContentServiceApi(Environment.Dev).getImages(
+        productEntity,
+        entityId,
+        ImageOrientation.Default,
+      );
+
+      images
+        .filter(image => image.screenSize === 'default')
```

Comment:
In this case `find` should be better because you should have only one default item